source/nvwave.py

```diff
@@ -158,13 +163,21 @@ def __init__(self, channels, samplesPerSec, bitsPerSample,
 		self._lock = threading.RLock()
 		self.open()
 
+	def _callOnMainThread(self, func, *args):

```

Comment:
Wouldn't this function be a candidate for inclusion in `core.py` beside callLater` you added there as of a59e007b35?

Reply:
This doesn't notify the watchdog, etc., so it's not safe for general usage. Normally, you'd use queueHandler.queueFunction to run something in the main thread, but I didn't want to deal with the overhead of the core pump, etc. for this case; it really should run ASAP.